btb/tuning/hyperparams/categorical.py

```diff
@@ -81,7 +88,7 @@ def _inverse_transform(self, values):
         if len(values.shape) == 1:
             values = values.reshape(1, -1)
 
-        return self._encoder.inverse_transform(values)
+        return self._encoder.inverse_transform(values.astype('object'))
```

Comment:
Does this changes address a bug? If so, it would be nice to also create an issue for this.